proto-lens-protoc/app/Proto/Google/Protobuf/Compiler/Plugin.hs

```diff
@@ -7,31 +7,31 @@ module Proto.Google.Protobuf.Compiler.Plugin (
         CodeGeneratorRequest(), CodeGeneratorResponse(),
         CodeGeneratorResponse'File(), Version()
     ) where
-import qualified Control.DeepSeq
-import qualified Data.ProtoLens.Prism
-import qualified Prelude
-import qualified Data.Int
-import qualified Data.Monoid
-import qualified Data.Word
-import qualified Data.ProtoLens
-import qualified Data.ProtoLens.Encoding.Bytes
-import qualified Data.ProtoLens.Encoding.Growing
-import qualified Data.ProtoLens.Encoding.Parser.Unsafe
-import qualified Data.ProtoLens.Encoding.Wire
-import qualified Data.ProtoLens.Field
-import qualified Data.ProtoLens.Message.Enum
-import qualified Data.ProtoLens.Service.Types
-import qualified Lens.Family2
-import qualified Lens.Family2.Unchecked
-import qualified Data.Text
-import qualified Data.Map
-import qualified Data.ByteString
-import qualified Data.ByteString.Char8
-import qualified Data.Text.Encoding
-import qualified Data.Vector
-import qualified Data.Vector.Generic
-import qualified Data.Vector.Unboxed
-import qualified Text.Read
+import qualified Data.ProtoLens.Runtime.Control.DeepSeq as Control.DeepSeq
```

Comment:
Is this right?

Reply:
Yeah, this is intentional.  Now that the module lives in proto-lens-protoc, it can depend on proto-lens-runtime (which reexposes the `Data.ProtoLens.Runtime.*` modules), the same as regular generated proto modules.